R/show_experiment.R

```diff
@@ -21,12 +21,20 @@ show_experiment <- function(file){
   if (length(exps) < 1)
     stop(paste0("File \"", file, "\" does not contain any experiment."))
   exps <- trimws(gsub("\\n+$", "", exps))
-  exp_info <- purrr::map(exps, function(x) {
-    if (str_detect(x, "type"))
+  exp_info <- purrr::map(exps, function(x){
```

Comment:
I think there should be a space between `)` and `{` no?

Reply:
Exact, I will write it back